pkg/gitreceive/k8s_util.go

```diff
@@ -12,11 +12,10 @@ import (
 )
 
 const (
-	slugBuilderName   = "deis-slugbuilder"
-	slugBuilderImage  = "quay.io/deisci/slugbuilder:v2-beta"
-	dockerBuilderName = "deis-dockerbuilder"
-	// change the value back to quay.io/deisci/dockerbuilder:v2-beta
-	dockerBuilderImage = "smothiki/dockerbuilder:v1.8"
+	slugBuilderName    = "deis-slugbuilder"
+	slugBuilderImage   = "quay.io/deisci/slugbuilder:v2-beta"
```

Comment:
This image path should be configurable ideally, not hard-coded. Let's fix that later.